lnrpc/lightning.proto

```diff
@@ -567,6 +567,10 @@ service Lightning {
     /* lncli: `subscribecustom`
     SubscribeCustomMessages subscribes to a stream of incoming custom peer
     messages.
+
+    To include messages with type outside of the custom range (>= 32768) lnd
```

Comment:
This description is a little ambiguous for someone unfamiliar with the spec.  Does ">= 32768" refer to types inside or outside the custom range?

```suggestion
    To include messages with type below the custom range (>= 32768) lnd
```

Reply:
I don't think that somebody unfamiliar with the spec _should_ be sending messages that are outside of the cutom range :') but will update the wording if I push again 👍